src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/ApprenticeshipProgrammes/ApprenticeshipProgrammesUpdateService.cs

```diff
@@ -2,75 +2,67 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Esfa.Recruit.Vacancies.Client.Application.Providers;
 using Esfa.Recruit.Vacancies.Client.Application.Services.ReferenceData;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Infrastructure.Exceptions;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.OuterApi;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.OuterApi.Requests;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.OuterApi.Responses;
 using Microsoft.Extensions.Logging;
 using Polly;
-using SFA.DAS.Apprenticeships.Api.Client;
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.ApprenticeshipProgrammes
 {
     public class ApprenticeshipProgrammesUpdateService : IApprenticeshipProgrammesUpdateService
     {
         private const int AcceptablePercentage = 10;
         private readonly ILogger<ApprenticeshipProgrammesUpdateService> _logger;
-        private readonly IStandardApiClient _standardsClient;
-        private readonly IFrameworkApiClient _frameworksClient;
+        private readonly IOuterApiClient _outerApiClient;
         private readonly IReferenceDataWriter _referenceDataWriter;
         private readonly IReferenceDataReader _referenceDataReader;
 
         public ApprenticeshipProgrammesUpdateService(
-            ILogger<ApprenticeshipProgrammesUpdateService> logger, 
-            IStandardApiClient standardsClient,
-            IFrameworkApiClient frameworksClient,
+            ILogger<ApprenticeshipProgrammesUpdateService> logger,
             IReferenceDataWriter referenceDataWriter,
-            IReferenceDataReader referenceDataReader)
+            IReferenceDataReader referenceDataReader,
+            IOuterApiClient outerApiClient)
         {
             _logger = logger;
-            _standardsClient = standardsClient;
-            _frameworksClient = frameworksClient;
             _referenceDataWriter = referenceDataWriter;
             _referenceDataReader = referenceDataReader;
+            _outerApiClient = outerApiClient;
         }
 
         public async Task UpdateApprenticeshipProgrammesAsync()
         {
-            var standardsTask = GetStandards();
-            var frameworksTask = GetFrameworks();
-
+            var trainingProgrammesTask = GetTrainingProgrammes();
```

Comment:
could save yourself a line of code and await this rather than WaitAll below

Reply:
👍🏻 changed